webapp/app/modules/userAuth/userAuth.constants.js

```diff
@@ -0,0 +1,2 @@
+export const ADD_WATER_FORM = 'add_water_form';
```

Comment:
wydaje mi się, że lepiej te stałe trzymać bezpośrednio przy formie, bo one nie do końca są związane z modułem autoryzacji. Czyli plik `addWaterForm.constants.js`

Reply:
Ja zasugerowałem takie rozwiązanie, było to związane z tym, że akcje reduxowe związane z tym formularzem znajdują się w tym module.
Może tak naprawdę wypadało by stworzyć osobny moduł tylko do akcji związanych z waterConsumption?
@wojciech-panek @mkmieciak co o tym myślicie ?
Pytanie na zasadzie, czy było by to lepsze rozwiązanie, niekoniecznie czy teraz rzeczywiście należy to robić.

Reply:
taka jest natura redux-forma, że można nim sterować z wielu miejsc, ale skoro to są pola bezpośrednio związane z formularzem, jak jego nazwa, czy nazwy poszczególnych pól to moim zdaniem najlepiej to trzymać bezpośrednio przy formularzu. Tworzenie osobnego modułu nie wydaje mi się dobrym pomysłem, bo właśnie o to chodzi, że moduły nam obrazują tzw. logikę biznesową, a komponenty/routy logikę widoków więc tworzenie sztucznego modułu pod widok (o ile nie jest to jakiś reużywalny element rozsiany w wielu miejscach apki) nie wydaje mi się dobrym pomysłem. W Adqusto mieliśmy grubą liczbę formularzy i takie podejście z cosntami się dobrze sprawdzało (było nawet niezbędne bo bez tego pojawiały się circular importy)

Reply:
Bardzo konkretna odpowiedź! Całkowicie mnie przekonała 👍